mlxtend/regressor/stacking_cv_regression.py

```diff
@@ -66,10 +66,14 @@ class StackingCVRegressor(BaseEstimator, RegressorMixin, TransformerMixin):
         be shuffled at fitting stage prior to cross-validation. If the `cv`
         argument is a specific cross validation technique, this argument is
         omitted.
+    store_train_meta_features : bool (default: False)
+        If True, meta-features for training data is stored when you `fit`
+        training data. You can get them by `self.train_meta_features_`.
```

Comment:
I suggest a minor rephrasing to

```
     store_train_meta_features : bool (default: False)
          If True, the meta-features computed from the training data used for fitting the
          meta-regressor stored in the `self.train_meta_features_` array, which can be
          accessed after calling `fit`. 
```

Could you also describe the shape/layout of the `self.train_meta_features_` array?